package-lock.json

```diff
@@ -31366,6 +31367,10 @@
       },
       "devDependencies": {
         "@types/glob": "^8.0.0",
+        "@types/marked-terminal": "^6.0.1",
+        "@types/minimist": "^1.2.2",
+        "@types/semver": "^7.3.9",
+        "@types/which": "^3.0.0",
```

Comment:
We don't ship TS types with this package so there's no need for these to be dependencies. `@types/marked-terminal` has a peer dependency on marked which was also causing limitations in the versions of marked installed via `npx`.